contribs/gmf/src/query/grid.scss

```diff
@@ -100,3 +100,9 @@ $table-height: $grid-height - $app-margin - 2 * $map-tools-size;
 .gmf-query-grid-active main {
   height: calc(100% - #{$grid-height});
 }
+
+.gmf-query-grid-too-many {
+  color: #ff0000;
```

Comment:
Please remove this `red` and use the bootstrap class `text-danger` to have the same red as other "error" messages.
Also, is red the right color ? perhaps a `text-warning` (yellow) is better ?

Suggestion: It looks better in italic to me.